src/App.js

```diff
@@ -1,24 +1,40 @@
-import logo from './logo.svg';
 import './App.css';
+import OrderForm from './OrderForm';
+import DropDown from './DropDown';
+import OrderImages from './OrderImages';
+import InstructionsForm from './InstructionsForm';
+import InstructionsList from './InstructionsList';
+
+import { useState } from 'react';
+
+
 
 function App() {
+
+  const [orderName, setOrderName] = useState('Customer #001');
+  const [drinkID, setDrinkId] = useState(1);
+  const [sideId, setSideId] = useState(1);
+  const [foodId, setFoodId] = useState(1);
+  const [instructions, setInstructions] = useState(['']);
```

Comment:
we don't need the empty string inside the square brackets. Just initializing with `useState([])` does the trick.